protocol/client/migrations/000002_add_unread_to_user_messages.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE user_messages
```

Comment:
is it worth it adding migration before some stable release?

Reply:
I think it's a nice exercise. Migrations did not work for me actually, I needed to upgrade our fork of https://github.com/golang-migrate/migrate.